sample-compose/src/main/java/com/squareup/radiography/sample/compose/RadiographyLogo.kt

```diff
@@ -0,0 +1,91 @@
+package com.squareup.radiography.sample.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Stack
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.WithConstraints
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.platform.DensityAmbient
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+import androidx.ui.tooling.preview.Preview
+import kotlin.math.roundToInt
+
+@Preview @Composable private fun RadiographyLogoPreview() {
+  RadiographyLogo()
+}
+
+/**
+ * Draws a Radiography logo, nesting the xray screen until it's too small to see.
+ */
+@Composable fun RadiographyLogo(modifier: Modifier = Modifier) {
+  val outerImage = imageResource(R.drawable.logo_outer)
+  val aspectRatio = outerImage.width.toFloat() / outerImage.height.toFloat()
+
+  Stack(modifier.aspectRatio(aspectRatio)) {
+    Image(outerImage)
+    InfiniteMirror(
+        centerOffsetFraction = Offset(.5f, .59f),
+        scaleFactor = .46f
+    ) {
+      Image(imageResource(R.drawable.logo_inner))
```

Comment:
imageResource should be pulled out of this lambda so it's not being loaded at every level.